plugins/modules/source_control/gitlab/gitlab_group.py

```diff
@@ -176,7 +212,10 @@ def createOrUpdateGroup(self, name, parent, options):
             changed, group = self.updateGroup(self.groupObject, {
                 'name': name,
                 'description': options['description'],
-                'visibility': options['visibility']})
+                'visibility': options['visibility'],
+                'project_creation_level': options['project_creation_level'],
+                'auto_devops_enabled': options['auto_devops_enabled'],
+                'subgroup_creation_level': options['subgroup_creation_level']})
```

Comment:
To shorten the diff the next time new options are added:
```suggestion
                'subgroup_creation_level': options['subgroup_creation_level'],
            })
```